base/reduce.jl

```diff
@@ -659,6 +659,19 @@ function count(pred, itr)
 end
 
 """
+    count(itr) -> Integer
+
+Count the number of `true` elements in `itr`.
+
+```jldoctest
+julia> count(4<=i<=6 for i in [2,3,4,5,6])
+3
+```
+"""
```

Comment:
I think this docstring could be combined with the two-argument version as is done in #20405.